frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
 
 Our goal is to reach a point where every view in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
 
 As such, the design system is still in its infancy and will grow with each release.
 
+## Purpose (what it is and what it's not)
+
+This Storybook-based design system documentation only seeks to describe the actual reusable components; it does not seek to document features where these components are used.
+
+The purpose of this document is to:
+
+- Have a single point of truth on what reusable components should look like, how they should behave, and what options they provide.
+- Allow product designers and front-end developers to have a shared understanding of how these common elements are specified and implemented
+- Allow QA a common base of reference when testing implementation of features using these common elements
```

Comment:
But as a "common base" or "generic point" to QA to check it makes sense to have this. If a feature introduces something new probably it will be documented by the team in the work package so it is already known by the QA tester.

Reply:
Changed to: "Allow QA a common base of reference when testing implementation of features that use these reusable components and patterns (and there is a doubt about a possible deviation)"